hclwrite/ast_block_test.go

```diff
@@ -414,7 +421,7 @@ func TestBlockSetLabels(t *testing.T) {
 		{
 			`foo "hoge" /* foo */  "" {}`,
 			"foo",
-			[]string{"hoge"},
+			[]string{"hoge", ""},
```

Comment:
I'm not sure why this test is covering the parsing of `foo "hoge" "" {}` as `foo "hoge" {}`, but it is an example of the bug we are trying to fix with this change, so I've updated the test. @minamijoyo, do you have any concerns about this test update?

Reply:
@alisdair I'm not sure why 🤔 
I checked the commit history and found that the test case was added by @apparentlymart 
https://github.com/hashicorp/hcl/pull/340/commits/7e5b148e1bda5f46dcbd1fc6ca727da3145a2f07

Reply:
🤦 Sorry, my mistake! I misread the commit history. Thanks!

Reply:
👌

Reply:
Hmm sorry about that! I imagine I must've misread the input while I was updating this (empty quotes in that position are unusual!) and managed to accidentally write a test that incorrectly passed, rather than a test that caught the bug. :confounded: